test/test_joint_infer.jl

```diff
@@ -1,5 +1,28 @@
 import JLD
+import ..Infer
 
+"""
+compute_obj_value
+computes obj value given set of results from one_node_infer and one_node_joint_infer
+"""
+function compute_obj_value(results,
+                           rcfs::Vector{RunCamcolField},
+                           stagedir::String;
+                           box=BoundingBox(-1000.,1000.,-1000.,1000.))
+    catalog, target_sources, neighbor_map, images = ParallelRun.infer_init(rcfs, stagedir;
+                                                                           box=box, objid="",
+                                                                           target_rcfs=RunCamcolField[])
+
+    ea = ElboArgs(images, [x["vs"] for x in results],
+                  Infer.get_sky_patches(images, catalog[target_sources]),
+                  collect(1:length(target_sources)), [1])
+    iter_count, obj_value, max_x, r = DeterministicVI.maximize_f(
+        DeterministicVI.elbo,
```

Comment:
Could you call `DeterministicVI.elbo` directly here to get the objective's value?